scripts/resolve_suggestions_urls.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python3
+
+import json
+from os import path, getcwd
+from jsonpath_rw import parse
+from jsonpointer import set_pointer
+
+UNFORMATTED_SUGGESTIONS_URL_ROOT = (
+    "https://cdn.eq.census-gcp.onsdigital.uk/data/{version}/{region}/{language}"
+)
+VERSION = "v4.0.0"
+
+
+def json_path_to_json_pointer(json_path):
+    """
+    Convert a json path string into a json pointer string.
+    :param json_path: the input data to search
+    :return: json pointer equivalent to the json path
+    """
+    json_pointer = json_path.replace("[", "").replace("]", "").replace(".", "/")
+    return f"/{json_pointer}"
+
+
+def resolve_schema(schema_filepath, suggestions_url_root=None):
```

Comment:
This will return an error if you run it twice independently. This should only try to resolve if it has something to resolve.